mdn/models.py

```diff
@@ -69,8 +69,8 @@ class FeaturePage(models.Model):
     status = models.IntegerField(
         help_text="Status of MDN Parsing process",
         default=STATUS_STARTING, choices=STATUS_CHOICES)
-    modified = ModificationDateTimeField(
-        help_text="Last modification time", db_index=True)
+    modified = models.DateTimeField(
+        help_text="Last modification time", db_index=True, auto_now=True)
```

Comment:
I dislike `auto_now` for its magic nature, and would rather set the value of the attribute in the `save` method explicitly